typescript/formsApps.d.ts

```diff
@@ -303,7 +305,13 @@ type _NewFormsApp = {
   organisationId: string
   /** Forms App progressive web app setting */
   pwaSettings: FormsAppPWASettings
+  /** Caching strategies for the Forms App */
+  cachingStrategies?: {
+    singleForm?: CacheStrategy
+    app?: CacheStrategy
+  }
   /** Forms App custom welcome email properties */
+
```

Comment:
```suggestion
```

And can we please make the prop on the config `_NewFormsApp['cachingStrategies']` instead of repeating it.